Kino/MainScreen/View/DetailsView/DetailsViewController.swift

```diff
@@ -72,7 +267,22 @@ class DetailsViewController: UIViewController {
         navigationItem.leftBarButtonItem = backButton
         
         scrollView.contentInsetAdjustmentBehavior = .never
-
+        
+        fillUIFromViewModel()
+    }
+    
+    func fillUIFromViewModel() {
```

Comment:
It would be better to use something like "setupUI"

Reply:
done